extensions/MESA/EGL_MESA_gl_texture_cubemap_image.txt

```diff
@@ -0,0 +1,106 @@
+Name
+
+    MESA_gl_texture_cubemap_image
+
+Name Strings
+
+    EGL_MESA_gl_texture_cubemap_image
+
+Contributors
+
+    Simon Zeni
+
+Contacts
+
+    Simon Zeni <simon@zeni.ca>
+
+Status
+
+    Draft
+
+Version
+
+    Version 0.1, 2022/08/04
+
+Number
+
+    EGL Extension 149
+
+Extension type
+
+    EGL client extension
+
+Dependencies
+
+    EGL 1.2 is required.
+
+    EGL_KHR_image_base and EGL_KHR_gl_image are required.
+
+    This extension is written against the wording of the EGL 1.2 Specification.
+
+    This extension requires an OpenGL or OpenGL ES client API supporting
+    texture cube maps, either in the core API or via extensions.
+
+Overview
+
+    This extension allows adds functionality to that provided by 
+    KHR_gl_image in order to support EGLImage cubemaps. It extends the
+    existing eglCreateImageKHR entry point from EGL_KHR_image_base.
+
+    For an overview of EGLImage operation, please see the EGL_KHR_image
+    specification.
+
+New types
+
+    None
+
+New functions
+
+    None
+
+New tokens
+
+    Accepted in the <target> parameter of eglCreateImageKHR:
+
+        EGL_GL_TEXTURE_CUBE_MAP_MESA                  PENDING
```

Comment:
The token is missing it's value because I don't see reserved values for MESA exts. Should I just use an arbitrary value?

Reply:
No. Please ask on the Mesa developer list if anyone is using / plans to use EGL enums in the range 0x3290-0x329F - those were reserved years ago but never actually put into an extension spec I'm aware of. I have no idea how Mesa manages its enum reservation blocks internally but if you can't establish one of those values is available, then we can allocate a new block.

Reply:
I've reached out on the ML last friday [1] and no answer since then. I'll make a pull request to allocate a new block.

[1]: https://lists.freedesktop.org/archives/mesa-dev/2022-August/225839.html

Reply:
I pushed the new block request in 32f9da23f1763f13f8a1b2f66831c96e9ffcbc77

Reply:
Mechanically this is fine, but the reservation needs to be merged to main branch ASAP to avoid potential collisions if anyone else needs a new block. Can you split it out as a separate PR? I can just accept that right away while the extension evolves.

Reply:
Sure, done in https://github.com/KhronosGroup/EGL-Registry/pull/168